### PR TITLE
update to new mdw

### PR DESCRIPTION
**Pull request by thepiwo**

### CODE REVIEW

File: utils/aeternity.js [hunk 1]
```diff
@@ -241,7 +238,7 @@ class Aeternity {
   }
 
   async getChainNames() {
-    return axios.get(`${MIDDLEWARE_URL}/middleware/names/active`).then(res => res.data).catch(this.logger.error);
+    return this.iterateMdw('names/active?limit=1000');
```
Comment (mradkov) [inline utils/aeternity.js:241]: Can we pass the limit as function argument?

Comment (thepiwo) [reply utils/aeternity.js:241]: why do we need this? the goal is to fetch all pages, so the limit doesn't matter

Comment (mradkov) [reply utils/aeternity.js:241]: Why is it there in the first place?

Comment (thepiwo) [reply utils/aeternity.js:241]: without limit you get 10 items per page, 1000 is the maximum limit

Comment (mradkov) [reply utils/aeternity.js:241]: Ok. So you iterate over the next 1000 (or what is left) in the iterateMdw fn. clear now

**Review by mradkov — APPROVED** [review]

**Review by mradkov — APPROVED** [review]

**Pull request merged**